plugins/outputs/sumologic/README.md

```diff
@@ -0,0 +1,68 @@
+in HTTP messages, encoded using one of the output data formats.
+Currently metrics can be sent using one of the following data formats, supported
+by Sumologic HTTP Source:
+
+  * `graphite` - for Content-Type of `application/vnd.sumologic.graphite`
+  * `carbon2` - for Content-Type of `application/vnd.sumologic.carbon2`
+  * `prometheus` - for Content-Type of `application/vnd.sumologic.prometheus`
+
+### Configuration:
+
+```toml
+# A plugin that can send metrics to Sumo Logic HTTP metric collector.
+[[outputs.sumologic]]
+  ## Unique URL generated for your HTTP Metrics Source.
+  ## This is the address to send metrics to.
+  # url = "https://events.sumologic.net/receiver/v1/http/<UniqueHTTPCollectorCode>"
+
+  ## Data format to be used for sending metrics.
+  ## This will set the "Content-Type" header accordingly.
+  ## Currently supported formats: 
+  ## * graphite - for Content-Type of application/vnd.sumologic.graphite
+  ## * carbon2 - for Content-Type of application/vnd.sumologic.carbon2
+  ## * prometheus - for Content-Type of application/vnd.sumologic.prometheus
+  ##
+  ## More information can be found at:
+  ## https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source/Upload-Metrics-to-an-HTTP-Source#content-type-headers-for-metrics
+  ##
+  ## NOTE:
+  ## When unset, telegraf will by default use the influx serializer which is currently unsupported
+  ## in HTTP Source.
+  data_format = "carbon2"
+
+  ## Timeout used for HTTP request
+  # timeout = "5s"
+  
+  ## HTTP method, one of: "POST" or "PUT". "POST" is used by default if unset.
+  # method = "POST"
+
+  ## HTTP Content-Encoding for writing request body. Can be set to "gzip" to compress the body.
+  ## This will set the "Content-Type" header accordingly.
+  # content_encoding = "gzip"
+
+  ## Max HTTP request body size in bytes before compression (if applied).
+  ## By default 1MB is recommended.
+  ## NOTE:
+  ## Bear in mind that in some serializer a metric even though serialized to multiple
+  ## lines cannot be split any further so setting this very low might not work
+  ## as expected.
+  # max_request_body_size = 1_000_000
+
+  ## Additional, sumologic specific HTTP header that can be set.
+  ## Full list can be found here:
+  ## https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source/Upload-Metrics-to-an-HTTP-Source#supported-http-headers
+  [outputs.sumologic.headers]
+    ## Desired source name
+    # X-Sumo-Name = ""       
+    ## Desired host name
+    # X-Sumo-Host = ""
+    ## Desired source category
+    # X-Sumo-Category  = ""
+    ## Comma-separated key=value list of dimensions to apply to every metric.
+    # X-Sumo-Dimensions = ""
+    ## Comma-separated, key=value list of metadata to apply to every metric.
+    # X-Sumo-Dimensions = ""
```

Comment:
nit: duplicate, i think this was supposed to be `X-Sumo-Metatags`. i would like to not show this header actually as we have long term plans to get rid of metatags in Sumo.

Reply:
done - removed